documentation/table_research.md

```diff
@@ -0,0 +1,73 @@
+# Table Research
+
+### Overview:
+- Project Table – stores information about projects that group monitors  
+- Uptime Monitor Table – stores monitor configurations and current state  
+- Uptime Log Table – stores logs of each check result for reporting and history 
+
+
+### Project Table:
+
+| Field | Type | Description |
+|--------|------|-------------|
+| id | UUID | Unique project identifier (partition key) |
+| name | String | This is the Project name |
+
+**Here is an example of what it could look like:**
+
+| id | name |
+|----|------|
+| `a12f...` | "Website Monitors" |
+| `b45d...` | "API Watcher" |
+
+
+### Uptime Monitor Table:
+
+| Field | Type | Description |
+|--------|------|-------------|
+| project_id | UUID | Partition key referencing Project |
+| id | UUID | Sort key (unique per monitor) |
+| name | String | Monitor name |
+| state | Enum | `new`, `up`, `down`, `paused` |
+| url | String | Endpoint to check |
+| frequency_mins | Number | How often to check (minutes) |
+| next_due_at | Timestamp | When next check is scheduled |
+| timeout_secs | Number | Request timeout (seconds) |
+| alert_settings | JSON | Slack/email alert configuration |
+| retry_config | JSON | Retry and realert intervals |

```

Comment:
This isn't consistent with the other docs. Please specify the reason for the change if there is one so we can discuss it.